server/www/templates/posts/views/category-select.html

```diff
@@ -0,0 +1,10 @@
+<fieldset ng-show="categories.length">
+    <legend translate="nav.categories">Categories</legend>
+
+    <div class="form-field checkbox" ng-repeat="(index, category) in categories">
+        <label>
+            <input checklist-value="category.id" checklist-model="model" type="checkbox" name="selectedCategories" > {{ ::category.name }}
```

Comment:
Yea thats a general failing of how angular scopes work. I had many issues - this way is working so far. But its possible I'll need to pass the entire `filters` object through or reference `$parent.model`